### ISSUE TITLE
V9: Cannot find module 'sanitize.css/page.css'

### ISSUE THREAD

**Issue by sylvainpolletvillard** [csstools/postcss-normalize#62]
Related: #60 

Can we also have a 9.0.1 patch please ? 

I'm stuck with PostCSS 7 in my project and it looks like postcss-normalize 9.0.0 now breaks every project build with this error:
```
Syntax Error: Error: Loading PostCSS Plugin failed: Cannot find module 'sanitize.css/page.css'
````

Thanks

**Comment by jonathantneal** [comment]
Yes, this will happen. Very likely this evening or tomorrow during a scheduled PostCSS plugins call at 1PM UTC.

**Comment by zrless** [comment]
When will you upgrade to version 9.0.1 ？

**Comment by huangd-d** [comment]
Can you add 9.0.1 tag; because my Project is 'postcss-normalize@9.0.0' in the file package.json. And i can not  rewrite the file.

**Comment by huangd-d** [comment]
> Can you add 9.0.1 tag; because my Project is 'postcss-normalize@9.0.0' in the file package.json. And i can not rewrite the file.

Can you add 9.0.1 tag; because my Project is 'postcss-normalize@9.0.0' in the file package.json. And i can not rewrite the file.

**Comment by zslucky** [comment]
same issue, pls freeze the dependency 😄

**Comment by romainmenke** [comment]
Does this issue still occur?

If so, can someone provide a minimal reproduction or submit a PR on this repo with a failing test?
